sdk/eventgrid/ci.yml

```diff
@@ -44,6 +44,11 @@ parameters:
   displayName: 'azure-messaging-eventgrid-cloudnative-cloudevents'
   type: boolean
   default: true
+- name: release_azureresourcemanagereventgrid
```

Comment:
I think this removal was intentional as it was moved to its own pipeline https://github.com/Azure/azure-sdk-for-java/blob/54c199dde1ef2bf738bcc838cdb6b87b9d2cf9c6/sdk/eventgrid/azure-resourcemanager-eventgrid/ci.yml